warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/MultiRFileOutputFormatter.java

```diff
@@ -557,15 +659,25 @@ public void write(BulkIngestKey key, Value value) throws IOException {
                     }
                 }
                 writer.append(key.getKey(), value);
-
             }
 
             @Override
             public void close(TaskAttemptContext context) throws IOException, InterruptedException {
-                // Close all of the Map File Writers
-                for (SizeTrackingWriter writer : writers.values()) {
+                for (Map.Entry<String,SizeTrackingWriter> entry : writers.entrySet()) {
+                    var writer = entry.getValue();
                     writer.close();
+                    if (loadPlanningEnabled) {
+                        var tableName = writerTableNames.get(entry.getKey());
+                        var rfilePath = usedWriterPaths.get(entry.getKey());
+                        addLoadPlan(tableName, writer.getLoadPlan(rfilePath.getName()));
```

Comment:
Getting NPEs on this line. 
Suggest adding a catch statement for additional debugging
```suggestion
                        addLoadPlan(tableName, writer.getLoadPlan(rfilePath.getName()));
                        try {
                            addLoadPlan(tableName, writer.getLoadPlan(rfilePath.getName()));
                        } catch (NullPointerException e) {
                            log.warn("Failed to find load plan: " + rfilePath + " for table: " + tableName);
                        }
```